config/webpacker.yml

```diff
@@ -82,6 +82,18 @@ test:
   # Compile test packs to a separate directory
   public_output_path: packs-test
 
+staging:
```

Comment:
Do we still need this?

Reply:
Nope, switch envs just before these meetings. Cleaning up.

Reply:
Dropped